test/Microsoft.DotNet.Tools.Tests.Utilities/PeReaderUtils.cs

```diff
@@ -11,6 +11,19 @@ namespace Microsoft.DotNet.Tools.Test.Utilities
 {
     public static class PeReaderUtils
     {
+        public static bool IsCrossgened(this PEReader peReader)
+        {
+            bool isCrossgened = false;
+
+            if (peReader.HasMetadata)
+            {
+                // 4 is the magic numbers that is set in the CLR header's flags when crossgened.
+                isCrossgened = (int)peReader.PEHeaders.CorHeader.Flags == 4;
```

Comment:
This field is a bit flag, so shouldn't do an equal comparison. Also, we have an enum that defines its values. Better to use something like:

`isCrossgened = (peReader.PEHeaders.CorHeader.Flags & CorFlags.ILLibrary) != 0;`

Unfortunately the term `ILLibrary` won't make sense to most people, so a comment that it means crossgened would be helpful.